python/pfs/drp/qa/utils/helpers.py

```diff
@@ -239,10 +240,10 @@ def getTargetType(arc_data, pfsConfig) -> pd.DataFrame:
 
 
 def getFitStats(
-    arcLines: ArcLineSet,
-    detectorMap: DetectorMap,
-    selection: np.ndarray[bool],
-    numParams: int = 0):
+        arcLines: ArcLineSet,
+        detectorMap: DetectorMap,
+        selection: NDArray[Bool],
```

Comment:
Does this work with the standard PFS stack? I would have thought our version of numpy was not sufficient.

Reply:
This is introduced in `1.20.0` and we are on `1.20.3` so just barely. It was passing scons testing.  I could change it to the string version to be a little safer.

Reply:
No, that's fine. I thought we were further behind than we actually are.

Reply:
I actually got the problem like this:

```
(lsst-scipipe-3.0.0) detectorMapQa.py --help
Traceback (most recent call last):
  File "/work/kiyoyabe/erun/run15/calibs/src/test/drp_qa/bin/detectorMapQa.py", line 2, in <module>
    from pfs.drp.qa.detectorMapQa import DetectorMapQaTask
  File "/work/kiyoyabe/erun/run15/calibs/src/test/drp_qa/python/pfs/drp/qa/detectorMapQa.py", line 27, in <module>
    from pfs.drp.qa.utils import helpers
  File "/work/kiyoyabe/erun/run15/calibs/src/test/drp_qa/python/pfs/drp/qa/utils/helpers.py", line 9, in <module>
    from numpy.typing import NDArray, Bool
ImportError: cannot import name 'NDArray' from 'numpy.typing' (/work/stack_INFRA-312/conda/miniconda3-py38_4.9.2/envs/lsst-scipipe-3.0.0/lib/python3.8/site-packages/numpy/typing/__init__.py)
```